imageio/imageio-dds/src/main/java/com/twelvemonkeys/imageio/plugins/dds/DDSHeader.java

```diff
@@ -0,0 +1,141 @@
+package com.twelvemonkeys.imageio.plugins.dds;
+
+import javax.imageio.IIOException;
+import javax.imageio.stream.ImageInputStream;
+import java.io.IOException;
+import java.nio.ByteOrder;
+import java.util.Arrays;
+
+public final class DDSHeader {
+
+	// https://learn.microsoft.com/en-us/windows/win32/direct3ddds/dx-graphics-dds-pguide
+	private int flags;
+	private int width;
+	private int height;
+	private int mipmap;
```

Comment:
Perhaps this field could be named `mipmapLevels` or `mipmapCount` or similar, to be more descriptive?